breadbox-client/README.md

```diff
@@ -1,15 +1,32 @@
-# The Breadbox Facade
+# The Breadbox Client
+
+A client library for accessing Breadbox
+
+Includes both an auto-generated `breadbox_client` module and a human-curated `breadbox_facade` which defines a more user-friendly interface for the client. 
+
+
+## Auto-Generating the breadbox_client
+In general, the `breadbox_client` directory is auto-generated in a github action. It is not checked in to the repo.
+
+If you would like to generate the client for local developmen/debugging, follow these steps:
+1. Install the `openapi-python-client` with the python environment manager of your choice. For example: `pipx install openapi-python-client==0.21.1`
```

Comment:
This is one of the things I wanted to get input on, but after thinking about it a bit more, I think you're right that the best option is to keep using the `./bb update-client` command for local development.

Historically, I've had some issues using that command without doing all of the following steps (and I wanted to ask if others have had similar issues): 
```
    cd breadbox-client-generator/
    poetry install && poetry shell
    cd ../breadbox
    ./bb update-client
```
However, I suspect that might have actually been an issue with my local python/poetry configuration, because the `./bb update-client` command does seem to be working for me now (and I'm not sure what changed). 

I am going to undo some of the changes I had made on this branch so we can keep using that command 👍

Reply:
FYI, I made the couple of changes we talked about. I also added a note to our check-in list so we remember to follow-up on potentially getting the `./bb update-client` command working in github actions. I'll wait to make any other changes to this branch until after we talk about that.